shiny/render/_render.py

```diff
@@ -560,3 +564,97 @@ def ui(
     ~shiny.ui.output_ui
     """
     return UiTransformer(_fn)
+
+
+# ======================================================================================
+# RenderDownload
+# ======================================================================================
+
+
+def download_button_wrapped(id: str) -> Tag:
+    return _ui.download_button(id, label=id)
+
+
+@output_transformer(
+    default_ui=download_button_wrapped
+)  # pyright: ignore[reportGeneralTypeIssues]
+async def DownloadTransformer(
+    _meta: TransformerMetadata,
+    _fn: DownloadHandler,
+    *,
+    button_label: Optional[TagChild] = None,
```

Comment:
I think this would work today with:
```suggestion
@output_transformer(
    default_ui=ui.download_button,
    default_ui_passthrough_args=["label"],
)  # pyright: ignore[reportGeneralTypeIssues]
async def DownloadTransformer(
    _meta: TransformerMetadata,
    _fn: DownloadHandler,
    *,
    label: Optional[TagChild] = "Download",
```
Definitely will be easier with Barret's new class-based approach.